server/src/main/resources/db/migration/updated natural textactions examples.sql

```diff
@@ -0,0 +1,192 @@
+UPDATE natural_text_action_examples
+SET example='In this example, execution will wait until specified text "\Register\" is present on current Page', workspace='https://travel.testsigma.com/login', `data`='{"test data": "Register"}', created_date=CURRENT_TIMESTAMP, updated_date=CURRENT_TIMESTAMP
+WHERE id=1;
+UPDATE natural_text_action_examples
+SET  example='In this example, execution will wait until specified text "\HOME123\" is absent on current Page', workspace='https://demoqa.com/', `data`='{"test data": "HOME123"}', created_date=CURRENT_TIMESTAMP, updated_date=CURRENT_TIMESTAMP
+WHERE id=2;
+UPDATE natural_text_action_examples
+SET  workspace='https://travel.testsigma.com/login', `data`='{"element": {"type": "id ", "value": "first_name "}, "test data": " test321"}', created_date=CURRENT_TIMESTAMP, updated_date=CURRENT_TIMESTAMP
+WHERE id=6;
+UPDATE natural_text_action_examples
```

Comment:
Have the Update statements in one line. This is to maintain the same coding style across the codebase. For e.g: 

```
UPDATE natural_text_action_examples SET  workspace='https://travel.testsigma.com/login', `data`='{"element":{"type": "id ", "value": "first_name "}, "test data": " test321"}', created_date=CURRENT_TIMESTAMP, updated_date=CURRENT_TIMESTAMP WHERE id=6;
```

Reply:
Got it, will make the changes.